tests/ui-fulldeps/codegen-backend/hotplug.rs

```diff
@@ -0,0 +1,20 @@
+//@ edition: 2021
+//@ build-pass
+//@ ignore-stage1 (requires matching sysroot built with in-tree compiler)
+
+//@ aux-codegen-backend: the_backend.rs
+//@ normalize-stdout-test: "libthe_backend.dylib" -> "libthe_backend.so"
+//@ normalize-stdout-test: "the_backend.dll" -> "libthe_backend.so"
+
+//@ revisions: normal dep bindep
+//@ compile-flags: --crate-type=lib -o -
+//@ [normal] compile-flags: --emit=link
+//@ [dep]    compile-flags: --emit=dep-info
+//@ [bindep] compile-flags: --emit=dep-info -Zbinary-dep-depinfo
```

Comment:
Unlike the original test, we no longer pass `--emit=link,dep-info`, because UI tests don't have a good way to check arbitrary output files (which is why we emit to stdout instead).

This does simplify the contents of the dep-info output a bit, but in a way that the original test wouldn't have noticed anyway, so I think it's fine.

Reply:
Oh, I did eventually figure out how to make this work. Each comma-separated value in `--emit` can have an output filename specified, so we can do e.g. `--emit=link,dep-info=-` to emit the link output to a file and the dep-info to stdout.

But since this PR is already approved, it may be easier to just change this in a separate PR.

Reply:
It's not in a rollup yet, so I think you can still make the change if you want to.